DANS-KNAW/easy-deposit
# EASY-1094: Files in the fetch.txt are resolved before the bag validity is checked

fixes EASY-1094
#### When applied it will

Resolve the files in fetch.txt (reads the files into the temporary bag directory). Now the bag contents can be validated.  When the files are not yet resolved, the bag is not valid.
#### How should this be manually tested?

In this package there are instructions about how to test using command line scripts. 

Note: to check the validity of the urls in the fetch.txt a regular expression is stored in application.properties.  Now we have a very permissive regular expression there, but that will be changed when we have decided on  what are acceptable urls. 

@janvanmansum 
- [x] Process code review comments
